virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,50 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
 
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+:FeatureName: Creating a virtual machine from an instance type
+include::snippets/technology-preview.adoc[]
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
 
-// |*Add volume*
-// |Upload a new volume or use an existing PersistentVolumeClaim (PVC).
+|*Volumes project*
+|The current namespace.
 
-// |*Filter* field
-// |Filter boot sources by operating system.
+|*Select volume to boot from*
+|Select a volume from the list to boot your VM from.
 
-// |Search field
-// |Search boot sources by keyword.
+|*Add volume*
+|Upload a new volume or use an existing PersistentVolumeClaim (PVC).
 
-// |*Select InstanceType*
-// |Select an instance type based on the number of CPUs and amount of memory available.
+|*Filter* field
+|Filter boot sources by operating system.
+
+|Search field
+|Search boot sources by keyword.
 
-// |*VirtualMachine details*
-// |Displays the current VM settings. You can change the name of the VM during this step.
+|*Select InstanceType*
+|Select an instance type based on the number of CPUs and amount of memory available.
 
-// |*Advanced settings*
-// |Add an authorized SSH key to the VM.
+|*VirtualMachine details* pane
+|Displays the current VM settings. You can change the name of the VM during this step.
 
-// |*Start this VirtualMachine after creation* checkbox
-// |Checked by default. Uncheck this to prevent the new VM from starting after creation.
+|*Advanced settings* pane
+|Add an authorized SSH key to the VM.
```

Comment:
I'm not sure about this. I think here is an "Authorized SSH key" section on the pane. It does not represent the full functionality of this pane (they will be adding dynamic key injection to this in the future), so it should not be used as the "definition".